mods/species/ascent/mobs/bodyparts.dm

```diff
@@ -88,3 +88,28 @@
 /obj/item/organ/external/head/insectoid/mantid/proc/reset_cooldown()
 	cooldown = FALSE
 	refresh_action_button()
+
+//Basically appends the crystal flag to all the types specified
+#define MANTIDIFY_ORGAN(ORGAN_PATH) ##ORGAN_PATH/Initialize(){BP_SET_CRYSTAL(src) . = ..();}
+//External organs
+MANTIDIFY_ORGAN(/obj/item/organ/external/chest/insectoid/mantid)
```

Comment:
What on earth? Why? Nothing else uses the mantid organs and this really isn't saving lines.

Reply:
The point is only have one line to change instead of ten million organ defines.